test/parallel_api/iterator/zip_iterator.pass.cpp

```diff
@@ -122,6 +124,34 @@ struct test_for_each
     }
 };
 
+struct test_for_each_structured_binding
```

Comment:
need to check this test only for C++17 configurations

Reply:
It can be wrapped in a corresponding macro.